lib/reek/examiner.rb

```diff
@@ -25,8 +25,8 @@ class Examiner
     #   The configuration for this Examiner.
     #
     # @public
-    def initialize(source,
-                   filter_by_smells = [],
+    def initialize(source:,
+                   filter_by_smells: [],
```

Comment:
You're right. On second thought I realize that this would make our API a little more verbose and for no apparent reason. 
I'll revert this change.